lesson1/js/getDatesDiff.js

```diff
@@ -1,4 +1,4 @@
-import { DateTime } from 'https://moment.github.io/luxon/es6/luxon.js'
+import { DateTime } from '../node_modules/luxon/src/luxon.js'
```

Comment:
можно покороче написать, webpack при сборке по умолчанию ищет модули в node_modules
```suggestion
import { DateTime } from 'luxon'
```